.github/workflows/on-pr-merge.yml

```diff
@@ -0,0 +1,29 @@
+name: Bump version and publish release
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - main
+
+jobs:
+  bump-version-publish-release:
+    runs-on: ubuntu-latest
+    if: "!contains( github.event.sender.login, 'broadbot')"
+    steps:
+      - name: Checkout current code
+        uses: actions/checkout@master
+        with:
+          token: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
+      - name: Bump tag and build version
+        uses: databiosphere/github-actions/actions/bumper@v0.0.3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          RELEASE_BRANCHES: main
+          VERSION_FILE_PATH: build.gradle
+          VERSION_LINE_MATCH: ^version
```

Comment:
you might want to put a word boundary on the right so it doesn't match `versionOptions` or something down the road.

Reply:
I think that's a good point, but going to leave it as is for now, to be consistent with other repos (stairway, common-lib).